example_yamls/example_tunet.yaml

```diff
@@ -2,9 +2,9 @@
 
 # I/O
 io_parameters:
-  data_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/rec20190524_085542_clay_testZMQ_8bit/20190524_085542_clay_testZMQ_
+  data_tiled_uri: http://0.0.0.0:8888/api/v1/metadata/reconstruction/rec20190524_085542_clay_testZMQ_8bit/20190524_085542_clay_testZMQ_
   data_tiled_api_key:
-  mask_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/mlex_store/mlex_segm_user/rec20190524_085542_clay_testZMQ_8bit/a558f7b1773855b9453bf6c91079699d
+  mask_tiled_uri: http://0.0.0.0:8888/api/v1/metadata/mlex_store/user1/rec20190524_085542_clay_testZMQ_8bit/pytest
```

Comment:
Just a note, with the updated Tiled server on SPIN, the clay dataset is now under: `https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/als_20190524_085542_clay/20190524_085542_clay_testZMQ_` and an example mask is at `https://tiled-seg.als.lbl.gov/api/v1/metadata/mlex_store/highres_spin/als_20190524_085542_clay/c35880e427239fb216ce73f3974ffe5e`

Reply:
Sounds good, captured and updated.